### PR TITLE
dispatch when covalent server is not available

### PR DESCRIPTION
**Pull request by madhur-tandon**
Fixes https://github.com/AgnostiqHQ/covalent/issues/1260

- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by madhur-tandon** [comment]
This should be ready to merge as well, @santoshkumarradha @kessler-frost @AlejandroEsquivel

**Comment by kessler-frost** [comment]
@madhur-tandon Again, thanks for the PR! Seems like the diff coverage isn't >=95%. These two lines (line 149, 150) in `covalent/_dispatcher_plugins/local.py` seem to be the culprit here:

```python
                r.raise_for_status()
                lattice_dispatch_id = r.content.decode("utf-8").strip().replace('"', "")
```

Once these are covered as well, i.e taking care of the `try` block, the diff coverage should be sufficient for passing the check.

File: covalent/_dispatcher_plugins/local.py [hunk 1]
```diff
@@ -141,12 +141,17 @@ def wrapper(*args, **kwargs) -> str:
...
-            r.raise_for_status()
-
-            lattice_dispatch_id = r.content.decode("utf-8").strip().replace('"', "")
+            lattice_dispatch_id = None
+            try:
+                r = requests.post(
+                    submit_dispatch_url, data=json_lattice, params={"disable_run": disable_run}
+                )
+                r.raise_for_status()
+                lattice_dispatch_id = r.content.decode("utf-8").strip().replace('"', "")
```
Comment (kessler-frost) [inline covalent/_dispatcher_plugins/local.py:150]: Tests for these.

**Review by kessler-frost — CHANGES_REQUESTED** [review]

File: covalent/_dispatcher_plugins/local.py (hunk 1 above)
Comment (madhur-tandon) [reply covalent/_dispatcher_plugins/local.py:155]: should be done now!

**Comment by madhur-tandon** [comment]
@kessler-frost last commit is to make `license/cla` work

**Comment by madhur-tandon** [comment]
Feel free to merge @kessler-frost!

**Review by kessler-frost — APPROVED** [review]

**Pull request merged**